.gitignore

```diff
@@ -13,3 +13,8 @@ dist/
 dump.rdb
 venv/
 .pytest_cache/
+
+bin/
+lib/
+node_modules/
```

Comment:
Why these?

Reply:
hmm, somehow these ended up in my project directory, not sure where does `node_modules` come from, I will remove it from .gitignore. `bin` and `libs` are some python auto-generated folders, you don't have them?